src/index.html

```diff
@@ -10,7 +10,50 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="page__body">
+    <form
+      data-qa="big"
+      class="search-bar search-bar--big"
+      action="#"
+      method="post"
+    >
+      <button type="submit" class="search-button search-button--big">
+        <img
+          src="images/Search.svg"
+          class="search-icon search-icon--big"
+          alt="search button"
+        >
+      </button>
+
+      <label>
+        <input
+          class="search-input search-input--big"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+    <form
+      data-qa="small"
+      class="search-bar search-bar--small"
+      action="#"
+      method="post"
+    >
+      <button type="submit" class="search-button search-button--small">
+        <img
+          src="images/Search.svg"
+          class="search-icon search-icon--small"
+          alt="search button"
+        >
+      </button>
+
+      <label>
+        <input
+          class="search-input search-input--small"
+          type="text"
+          placeholder="Los Angeles"
```

Comment:
```suggestion
          placeholder="Try &quot;Los Angeles&quot;"
```

![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/70009205/7d4ffd56-b578-418b-b3b2-df438f3d46ab)